src/components/Guilds/ProposalStatus/index.tsx

```diff
@@ -1,10 +1,11 @@
 import styled from 'styled-components';
 import { Box } from '../common/Layout';
+
 const Status = styled.div`
   display: flex;
   flex-direction: row;
   width: 100%;
-  justify-content: flex-end;
+  justify-content: flex-start;

```

Comment:
changing this value to start, is causing this issue on the proposals page, where the status button is squishing up the text
![image](https://user-images.githubusercontent.com/53880692/144121394-4724fa5f-4a89-4b9d-9226-26067b703073.png)

Reply:
nice!